### DISCUSSION TITLE
Don't forget to only mark the latest release as such for Github release creation

### DISCUSSION THREAD

**Discussion by andytson-inviqa** [argoproj/argo-cd#15244] [General]
It's showing 2.5.22 as the latest release, but 2.8.2 is the latest release.

When you create Github Releases, you should only have "Set as the latest release" ticked for the latest bugfix release of the latest minor version. Github doesn't have a concept of a latest bugfix version for each minor release.

![Screenshot 2023-08-25 at 11 27 27](https://github.com/argoproj/argo-cd/assets/11387086/1a9bf6d2-330c-498f-b30c-37a8582cd091)

![Screenshot 2023-08-25 at 17 14 41](https://github.com/argoproj/argo-cd/assets/11387086/6d170681-2a80-414e-9aad-ec6fa3ed8ac9)

**Comment by andytson-inviqa** [comment]
sorry the first screenshot was the "latest" release before you released 2.5.22. Github shows 2.5.22 now

**Comment by crenshaw-dev** [comment]
Created an issue to track the fix: https://github.com/argoproj/argo-cd/issues/15245
